Cargo.toml

```diff
@@ -9,15 +9,15 @@ readme = "README.md"
 license-file = "LICENSE"
 keywords = ["lsm", "storage-engine", "embedded", "persistence"]
```

Comment:
Piggy-backing on your changes, could we enrich/update this metadata as follows?:

`categories = ["database"]`
`license = "Apache-2.0"` -> instead of `license-file`.
`documentation = "https://docs.rs/lsmlite-rs/"`

Reply:
Done. I have also sorted the keys by name.